include/openssl/macros.h

```diff
@@ -14,9 +14,6 @@
 #include <openssl/opensslv.h>
 
 
-/* Helper macros for CPP string composition */
-# define OPENSSL_MSTR_HELPER(x) #x
-# define OPENSSL_MSTR(x) OPENSSL_MSTR_HELPER(x)
```

Comment:
What does MSTR even mean? Definitely the thing I'm going to be looking for when I'm in need of this. I might have found this if they had called it OPENSSL_STRINGIFY...

Reply:
"make string"?  Beats me....